DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: periscope2
 Type: Package
 Title: Enterprise Streamlined 'shiny' Application Framework Using 'bs4Dash'
-Version: 0.1.3
+Version: 0.1.3.9000
```

Comment:
The version should be 0.1.4.9000, this is the development (.9000) version of the next package

Reply:
done